homeassistant/components/netgear_lte/__init__.py

```diff
@@ -45,6 +44,28 @@
 
 EVENT_SMS = "netgear_lte_sms"
 
+ALL_SENSORS = [
+    "sms",
+    "sms_total",
+    "usage",
+    "radio_quality",
+    "rx_level",
+    "tx_level",
+    "upstream",
+    "connection_text",
+    "connection_type",
+    "current_ps_service_type",
+    "register_network_display",
+    "current_band",
+    "cell_id",
+]
+
+ALL_BINARY_SENSORS = [
+    "roaming",
+    "wire_connected",
+    "mobile_connected",
+]
```

Comment:
Would it not be better to have the sensor names as constants in const.py, especially because they are used in multiple places?

Reply:
Hmm, good point. The reason I did not do this was because this will be removed in about 6 months.

Reply:
Ah, I now understand that it is only for the import of YAML to the config flow. Then it is fine for me :blush: